keep/api/core/demo_mode_runner.py

```diff
@@ -0,0 +1,417 @@
+        description="Production Queue",
+        team="keep",
+        email="support@keephq.dev",
+        slack="https://slack.keephq.dev",
+        ip_address="10.0.0.4",
+        mac_address="",
+        category="Kafka",
+        manufacturer="",
+        dependencies={
+            "processor": "AMQP",
+        },
+        application_ids=[],
+        updated_at="2024-11-18T09:31:31"
+    ),
+    TopologyServiceInDto(
+        source_provider_id="Prod-Datadog",
+        repository="keephq/keep",
+        tags=[],
+        service="processor",
+        display_name="Processor",
+        environment="prod",
+        description="Processing Service",
+        team="keep",
+        email="support@keephq.dev",
+        slack="https://slack.keephq.dev",
+        ip_address="10.0.0.5",
+        mac_address="",
+        category="go",
+        manufacturer="",
+        dependencies={
+            "storage": "HTTP/S",
+        },
+        application_ids=[],
+        updated_at="2024-11-18T10:02:20"
+    ),
+    TopologyServiceInDto(
+        source_provider_id="Prod-Datadog",
+        repository="keephq/keep",
+        tags=[],
+        service="backoffice",
+        display_name="Backoffice",
+        environment="prod",
+        description="Backoffice UI to control configuration",
+        team="keep",
+        email="support@keephq.dev",
+        slack="https://slack.keephq.dev",
+        ip_address="172.1.1.0",
+        mac_address="",
+        category="nextjs",
+        manufacturer="",
+        dependencies={
+            "api": "HTTP/S",
+        },
+        application_ids=[],
+        updated_at="2024-11-18T10:11:31"
+    ),
+    TopologyServiceInDto(
+        source_provider_id="Prod-Datadog",
+        repository="keephq/keep",
+        tags=[],
+        service="storage",
+        display_name="Storage",
+        environment="prod",
+        description="Storage Service",
+        team="keep",
+        email="support@keephq.dev",
+        slack="https://slack.keephq.dev",
+        ip_address="10.0.0.8",
+        mac_address="",
+        category="python",
+        manufacturer="",
+        dependencies={},
+        application_ids=[],
+        updated_at="2024-11-18T10:13:56"
+    )
+]
+
+application_to_create = {
+    "name": "Main App",
+    "description": "It is the most critical business process ever imaginable.",
+    "services": [
+        {"name": "API Service", "service": "api"},
+        {"name": "DB", "service": "db"},
+        {"name": "Kafka", "service": "queue"},
+        {"name": "Processor", "service": "processor"},
+        {"name": "Storage", "service": "storage"}
+    ]
+}
+
+def get_or_create_topology(keep_api_key, keep_api_url):
+    services_existing = requests.get(
+        f"{keep_api_url}/topology",
+        headers={"x-api-key": keep_api_key},
+    )
+    services_existing.raise_for_status()
+    services_existing = services_existing.json()
+
+    if len(services_existing) == 0 or True:
```

Comment:
because of "or True" it'll always run, I think

Reply:
gooooooood catch!